extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/duplicate/DuplicateResourceWarningTest.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.resteasy.reactive.server.test.duplicate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+import java.util.stream.Collectors;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+
+public class DuplicateResourceWarningTest {
+
+    @RegisterExtension
+    static QuarkusUnitTest runner = new QuarkusUnitTest()
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addClasses(GreetingResource.class, GreetingResource2.class, GreetingResource3.class))
+            .overrideConfigKey("quarkus.resteasy-reactive.fail-on-duplicate", "false")
```

Comment:
So far these properties have been prefixed with `quarkus.rest` instead of `quarkus.resteasy-reactive`. Let's keep it like that for consistence

Reply:
I didn't changed the config prefic, it's fixed by `ResteasyReactiveConfig` class

Reply:
Yup: https://quarkus.io/guides/all-config#quarkus-resteasy-reactive-common_quarkus-resteasy-reactive-common-resteasy-reactive-common